.github/workflows/ci.yml

```diff
@@ -373,6 +373,14 @@ jobs:
         shell: bash
         run: echo OPENSSL_SRC_PERL=C:/Strawberry/perl/bin/perl >> $GITHUB_ENV
 
+      - name: Additional Check
```

Comment:
This check is quiet costly: we re-compile the entire project and dependencies (and we add pressure on the cache)

Instead what we should do:
- modify `.github/actions/paths-filter` to add a new `rust-platform` entry this match `oxidation/libparsec/crates/platform_*/**`[^1]
- add this `rust-platform` to the `client-web` filter

Now `🌐 Web tests` will be triggered any time the platform code is modified ;-)

[^1]: we can also be a bit more cunning in filtering and have instead a `rust-platform-web` that match on `oxidation/libparsec/crates/platform_*/src/web/**`, this way we avoid triggering the web build if changing the native part

Reply:
Done